SideBar.py

```diff
@@ -90,7 +90,73 @@ def copy(self, source, destination):
 			shutil.copy2(source, destination)
 
 		if self.view:
-			self.view.erase_status('ZZZ')
+			self.view.erase_status('SideBarTools:Copy')
 
 	def description(self):
 		return 'Duplicate File…'
+
+
+def temporary_status_message(view, message, key='SideBarTools', duration=5000):
```

Comment:
Ok, I did some quick tests and turns out messages set with `View.set_status` do not disappear automatically, but `sublime.status_message` and `Window.status_message` do (apparently `sublime.status_message` finds the active window and then calls `status_message` on it).

On the other hand, `View.set_status` is specific only to the view; this means, if we switch tabs, the message disappears until we switch back to that view. So the question is, do we want temporary, window-level messages, or permanent messages (controlled by us, we can make them disappear).

For this plugin specifically I think is best to use `Window.status_message`, so tab-switching doesn't hide the message, specially since (by default) almost every click in the sidebar will create a new view (transient or not).